tests/README.md

```diff
@@ -90,25 +106,17 @@ A basic comparison function is also provided for checking the contents of report
 easily be adapted for testing other types of text based files.  
 
 
-Common Problems
----------------
+## Common Problems
 
 When adding new apps and tests the json format can be a bit finicky and json parsing errors 
 aren't being reported in a user friendly manner. 
 
-Sometimes the outputapi.dll can not be found at run time. The dll can not be accessed when 
-swmm_reader package is installed as a compressed python egg. The egg needs to be uncompressed. 
-
-Another reason the dll may not be found is that the absolute path to the dll location is 
-hard coded on line 569 the outputapi.py file in the swmm_reader package. On Windows systems 
-the path needs to point to the installed location of the package for the dll to load properly. 
-
 The absolute path to the executable must be specified in the json file describing the application
 for testing. 
 
 SWMM requires the absolute path of data files referenced in a SWMM input file (e.g. rain gauge data). 
 
-The packages swmm_reader and nrtest_swmm were developed and run on Windows with 64 bit version 
+The packages swmm_output and nrtest_swmm were developed and run on Windows with 64 bit version 
```

Comment:
It seems like we should probably move these packages to separate repositories ?

Reply:
Right now my highest priority is automating the Windows build and getting the regression tests passing. All in due time.